src/reconstruct_loopset.jl

```diff
@@ -176,10 +176,7 @@ function add_mref!(
     sptrs::Expr, ls::LoopSet, ar::ArrayReferenceMeta, @nospecialize(_::Type{Ptr{T}}),
     C::Int, B::Int, sp::NTuple{N,Int}, name::Symbol
 ) where {T,N}
-    Tsym::Symbol = get(VectorizationBase.JULIA_TYPES, T) do
-        Symbol(T)
-    end
-    add_mref_ptr!(sptrs, ls, ar, Tsym, C, B, sp, name)
+    add_mref_ptr!(sptrs, ls, ar, T, C, B, sp, name)
```

Comment:
The purpose of this method was to provide a function barrier. You can delete it and rename `add_mref_ptr!`